import/popups/ContextMenu.qml

```diff
@@ -211,6 +211,17 @@ ContextMenuInterface {
                 onClicked: root._hide()
             }
 
+            MenuItem {
+                visible: root.isImage && root.url
+                //: Copy image address to clipboard from context menu
+                //% "Copy image address"
```

Comment:
Just "Share" as the context indicates already that you long pressed on top of an image.

Reply:
You could enhance WebShareAction to take option share menu title. Currently it's always "Share link".

See: 
https://github.com/sailfishos/sailfish-components-webview/pull/155/files#diff-9452e87a299d714c5663501930922ad070ab060fc3b90f654317fac5bacc7ba4R146